tests/vm_test.go

```diff
@@ -1957,24 +1957,37 @@ status:
 
 	Context("[Serial] when node becomes unhealthy", Serial, func() {
 		const componentName = "virt-handler"
+		var nodeName string
+
+		JustAfterEach(func() {
```

Comment:
`AfterEach` is enough here. Inner `AfterEach` are executed before the outer one

Reply:
Done.